lab-dan/test/app-test.js

```diff
@@ -0,0 +1,11 @@
+var expect = require('chai').expect;
```

Comment:
Nice! This is great, requiring your app, and assigning mock data to the cli args.